scipy/stats/stats.py

```diff
@@ -3736,13 +3781,25 @@ def pearsonr(x, y):
         Input array.
     y : (N,) array_like
         Input array.
+    alternative : {'two-sided', 'greater', 'less'}, optional
+        Indicates the alternative hypothesis. The default value is
+        'two-sided'.
+    confdence_level : float, optional
+        The confidence level for the calculation of the confidence
```

Comment:
I misread this the first time. Would `The confidence level for the calculation of the correlation coefficient confidence interval` be any better? Just

Reply:
Done.